File: 01_Mon/day1.py
# Ask the user to provide a positive integer (x)
# For each number (n) less than x, print whether or not the number is even or odd.

num = int(input("Enter a number: "))

# for n in range(num):
#     if n != 0:
#         if n % 2:
#             print(f'{n} : odd')
#         else:
#             print(f'{n} : even')

i = 1
while (i < num):
    if i != 0:
        if i % 2 == 0:
            print(f" {i} : Even")

        else:
            print(f" {i} : Odd")
    i += 1

File: 01_Mon/day1.js
function evenOdd() {
    var startNum = [1, 2, 3, 4, 5, 6, 7, 8];
    startNum.forEach(function (e) {
        if (e != 0) {
            if (e % 2 == 0) {
                console.log(e + ": Even")
            } else {
                console.log(e + ": Odd")
            }
        }
    })
}

// function evenOdd() {
//     num = 24
//     startNum = 1;

//     while (startNum < num) {
//         if (startNum != 0) {
//             if (startNum % 2 == 0) {
//                 console.log(`${startNum} : Even`)
//             } else {
//                 console.log(`${startNum} : Odd`)
//             }
//         }
//         startNum += 1;
//     }



// }

evenOdd()
